Task Liverpool/src/main/java/pages/HomePage.java

```diff
@@ -0,0 +1,27 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class HomePage extends BasePage {
+
+    private By searchBar =By.id("mainSearchbar");
+    private By searchBarButton = By.cssSelector(".input-group-text .icon-zoom");
+
+    public HomePage(WebDriver webDriver) {
+        super(webDriver);
+    }
+
+    public void clickSearchBar(){
+        click(searchBar);
+    }
+
+    public void typeSearchBar(){
+        type(searchBar, "1103656130");
```

Comment:
you should receive the string as a parameter, **do not hardcode anything**

Reply:
agree, fixed.